src/Device/Parser.cpp

```diff
@@ -307,9 +307,13 @@ NMEAParser::GSA(NMEAInputLine &line, NMEAInfo &info)
    */
 
   line.Skip();
-
-  if (line.Read(0) == 1)
-    info.location_available.Clear();
+  
+  // determine fix type
+  unsigned fix_mode;  
```

Comment:
The comment says "fix type", but the variable is named "fix_mode". Huh?
Why even the code comment, it only states the obvious (if its nomenclature wouldn't contradict with the actual code).